packages/runner/lib/exec.ts

```diff
@@ -192,7 +215,7 @@ export async function exec(
                     return {
                         success: false,
                         error: {
-                            type: 'script_http_error',
+                            type: 'script_network_error',
```

Comment:
why changing the error code?

Reply:
Previously we weren't including any payload information, so it made sense to have the same error code on a network error or an error where we actually received an HTTP response. Once we're including response information it makes sense to have a case where you can expect response data to be included with the error (`script_http_error`) and a case where you can't (`script_network_error`).

Reply:
does not data always means network error though?

Reply:
Oh right, we should check on no response, not no data. Fixing now!

Reply:
@TBonnin See 3e2cb1b6e. Requesting review since this was a little non-trivial.